ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -276,9 +276,7 @@ QString ManageTokensController::settingsGroupName() const
 
 bool ManageTokensController::hasSettings() const
 {
-    Q_ASSERT(!m_settingsKey.isEmpty());
-    const auto groups = m_settings.childGroups();
-    return groups.contains(settingsGroupName());
+    return m_settingsData.size() > 0;
```

Comment:
```suggestion
    return !m_settingsData.isEmpty();
```

Reply:
You don't need to count them all :)

Reply:
Haha, forgot it's a `QHash`, so there's no difference :)

There would be though in case of a `QList`:
```cpp
inline bool isEmpty() const noexcept { return d->end  == d->begin; }
```